tests/devices/i19/test_unit_hutch_conditional_shutter.py

```diff
@@ -0,0 +1,39 @@
+from unittest import TestCase
+
+import pytest
+
+from dodal.devices.i19.shutter import (
+    HutchConditionalShutter,
+    HutchInvalidError,
+    HutchState,
+)
+
+
+async def unit_test_shutter_created_without_raising_errors(
```

Comment:
Could: This actually need not be an `async` function. To a first approximation an `async` function is only needed if you are calling other `async` functions from it. The constructor of `HutchConditionalShutter` is synchronous and so this isn't needed. If you were calling the `set` here it would be needed. Does no harm though so not the end of the world to leave it in.